nxcomp/Loop.cpp

```diff
@@ -9340,6 +9341,100 @@ int ParseCommandLineOptions(int argc, const char **argv)
 
           return -1;
         }
+        case 'd':
+        {
+          if ( argi+1 >= argc )
+          {
+            PrintUsageInfo(nextArg, 0);  
+            return -1;
+          }
+
+          int level = 0;
+	  level = strtol(argv[argi+1], NULL, 10);
+
+	  if ( errno && (level == 0) )
+	  {
+            cerr << "Warning: Failed to parse log level. Ignoring option." << std::endl;
+          }
+	  if ( level < 0 )
+	  {
+            cerr << "Warning: Log level must be a positive integer. Ignoring option." << std::endl;
```

Comment:
Hm, you're not really ignoring it, but setting the level to a negative value the user passed. That probably will lead to no log output at all, judging from the other code. Might want to explicitly set level to `NXWARNING` here instead, which is supposed to be the general default?